docs/BindingWithGrammar.md

```diff
@@ -0,0 +1,114 @@
+# Binding with Grammar
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides support for automatic XML schema/grammar binding through generation and/or using an existing `.xsd/.dtd` file.
+
+[Validation with XSD grammar](Validation.md#validation-with-xsd-grammar) and [Validation with DTD grammar](Validation.md#validation-with-dtd-grammar) provide more information on manual binding of an XML document.
+
+In the following sections, we would like to bind the XML document `foo.xml`:
+
+```xml
+<foo>
+  <bar />
+</foo>
+```
+
+with an associated grammar file.
+
+## Binding with Existing Grammar
+
+Consider an XSD document, `foo.xsd` file (in the same directory as foo.xml) defined as follows :
+
+```xsd
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <xs:element name="foo">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element name="bar" />
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+</xs:schema>
+```
+
+This file (or a corresponding `.dtd` file) can be bound using a [binding wizard](#the-xml-binding-wizard), which can be triggered by [Command](#command), [CodeLens](#codelens) or [Quick Fix/CodeAction](#quick-fix/codeaction).
```

Comment:
The link to "Quick Fix/CodeAction" here also doesn't work in VS Code

Reply:
The link seems to work without a change, could you confirm?

Reply:
It doesn't seem to work for me on Linux or windows on the latest stable VS Code. I think maybe rewording the title to avoid the `/` should work?